website/server/controllers/api-v3/groups.js

```diff
@@ -301,6 +302,8 @@ api.joinGroup = {
 
     group.memberCount += 1;
 
+    if (group.purchased.plan.customerId) await payments.updateGroupPlan(group);
```

Comment:
question: when the group is deleted or created the stripe plan is automatically deleted / created?